### DISCUSSION TITLE
Welcome to one-tap-google-sign-in Discussions!

### DISCUSSION THREAD

**Discussion by surendhar153** [surendhar153/one-tap-google-sign-in#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mk24x7** [comment]
Hi
This seems like a really neat plugin, but I'd like to add the option to display the one-tap widget on all pages of the website. How do we do that?

I'd like to contribute by making a video / writing steps to get a client ID for the plugin. It'll motivate more people to adopt this plugin from the WordPress repository.
